src/containers/MainDashboard/MainDashboard.tsx

```diff
@@ -58,19 +78,19 @@ const ADD_ROADMAP = gql`
 const MainDashboard: React.FC = () => {
   const client = useApolloClient();
   const [titleInput, setTitleInput] = useState('');
-  const [selectionInput, setSelectionInput] = useState('');
+  const [selectionInput, setSelectionInput] = useState('IT');
   const [flag, setFlag] = useState(false);
   // get userID from cache
-  const userID: IUserID | any = client.cache.readQuery({ query: GET_USER_ID });
+  const token: any = localStorage.getItem('token');
+  const { id } = jwtDecode(token);
```

Comment:
This operation is strictly coupled to the main dashboard, could be a better option splitting this logic and bringing it into an authentication service.